web/src/stats/stats.ts

```diff
@@ -605,8 +695,8 @@ function populate_messages_sent_by_client(data) {
         },
     };
 
-    let user_button = "everyone";
-    let time_button;
+    let user_button: "everyone" | "user" = "everyone";
+    let time_button: "cumulative" | "year" | "month" | "week";
```

Comment:
We can extract some quick Zod schemas for these literals.

Reply:
Did. That's a great suggestion